classes/class-wc-connect-api-client.php

```diff
@@ -193,6 +193,7 @@ protected function request( $method, $path, $body = array() ) {
 				'last_heartbeat' => get_option( 'wc_connect_last_heartbeat', 0 ),
 				'last_rate_request' => get_option( 'wc_connect_last_rate_request', 0 ),
 				'active_services' => $this->wc_connect_loader->get_active_services(),
+				'enable_stats' => ! Jetpack::is_staging_site(),
```

Comment:
I guess it's a bit of a personal opinion, but I think it almost always makes sense to use the one that is not the "most common scenario". In this case, disabling stats is the exception not the rule, thus checking for `disable_stats` makes more sense. If we only wanted stats for certain requests, then I'd agree with you.

---

Filter: I mean a WordPress filter (`apply_filters( ... )`) so that someone (mostly us) can toggle this as needed.